flytekit/clients/auth_helper.py

```diff
@@ -90,7 +90,7 @@ def get_authenticator(cfg: PlatformConfig, cfg_store: ClientConfigStore) -> Auth
         return DeviceCodeAuthenticator(
             endpoint=cfg.endpoint,
             cfg_store=cfg_store,
-            audience=cfg.audience,
```

Comment:
We made it so that DeviceCodeAuthenticator sets `self._audience = cfg.audience` within `__init__` in its class definition rather than manually passing it in `auth_helper`

We can revert it back, but we thought it would clean up the code but perhaps it was an un-intuitive move.